lib/iris/fileformats/netcdf/loader.py

```diff
@@ -236,7 +237,19 @@ def _get_cf_var_data(cf_var, filename):
             )
             # Get the chunking specified for the variable : this is either a shape, or
             # maybe the string "contiguous".
+            if CHUNK_CONTROL.mode == CHUNK_CONTROL.Modes.AS_DASK:
```

Comment:
Enumeration checks should be `is`, rather than `==` - it's the actual same object rather than just an equivalent one.